omod/src/main/webapp/resources/htmlforms/HMIS-003-HealthEducationPage.xml

```diff
@@ -0,0 +1,188 @@
+<htmlform formName="HMIS 003 HIV Care ART Card - Health Education And Psychosocial Form"
+          formDescription="Captures health eduction and counseling encounters"
+          formEncounterType="6d88e370-f2ba-476b-bf1b-d8eaf3b1b67e"
+          formUuid="b8224534-7df0-4c00-ba8e-6b1f291087eb"
+          formVersion="1.2"
+          formAddMetadata="yes"
+          formUILocation="patientDashboard.visitActions"
+          formOrder="2"
+          formIcon="icon-medkit"
+          formShowIf="(visit.active || !visit.active) &amp;&amp; patient.person.dead==false &amp;&amp; sessionLocation.uuid=='86863db4-6101-4ecf-9a86-5e716d6504e4'"
+          formDisplayStyle="Standard"
+          formLabel="ART Card -  Health Education And Psychosocial Support Form">
+
+
+
+
+    <div class="card">
+        <div class="headers" style="text-align: center;">
+            <h3 style="padding: 10px;">HMIS-003 Health Education And Psychosocial Support Form</h3>
+        </div>
+
+
+        <div class="card">
+            <div class="card-header">
+                Visit details
+            </div>
+            <div class="card-body">
+                <div class="row">
+                    <div class="col-6">
+                        <div class="row">
+                        Visit Date  <encounterDate default="today"  disallowMultipleEncountersOnDate="block" />
+                        </div>
+                    </div>
+                    <div class="col-">
+                        Is Visit On Schedule <obs id="br" conceptId="90069" style="checkbox"/>
+
+                    </div>
+                </div>
+
+            </div>
+
+        </div>
+        <div class="card-body">
+        </div>
+        <div class="card">
+            <div class="card-header">
+                Encounter Information
+            </div>
+
+            <div class="card-body">
+                <table class="table table-striped">
+                    <tr>
+                        <td>Settings</td>
+                        <td>
+                            <div class="row">
```

Comment:
@solemabrothers Remove div class="row" in all table rows

Reply:
@ssmusoke  This has been done, however, i had to include an internal styling code for the alignment of radio buttons